etip/trackers/management/commands/compare_with_exodus.py

```diff
@@ -0,0 +1,95 @@
+from django.core.management.base import BaseCommand, CommandError
+import requests
+
+from trackers.models import Tracker
+
+EXODUS_API_BASE_URL = 'https://reports.exodus-privacy.eu.org'
+EXODUS_API_BASE_PATH = '/api/trackers'
+FIELDS_TO_LOOKUP = [
+    'name',
+    'code_signature',
+    'network_signature',
+    'website'
+]
+LOOKUP_IS_EXACT = 'EXACT'
+LOOKUP_IS_DIFFERENT = 'DIFFERENT'
+LOOKUP_MULTIPLE = 'MULTIPLE_IN_ETIP'
+LOOKUP_NOT_FOUND = 'NOT_FOUND_IN_ETIP'
+
+
+class Command(BaseCommand):
+    help = 'Compare trackers stored in Exodus with trackers from ETIP DB'
+
+    def handle(self, *args, **options):
+        exodus_trackers = self.get_all_from_exodus()
+        self.stdout.write(
+            'Retrieved ' + str(len(exodus_trackers)) + ' tracker from Exodus '
```

Comment:
Please use format for printing logs, example here:
`'Retrieved {} trackers from Exodus'.format(len(exodus_trackers))`

See https://pyformat.info/

Reply:
Thanks, updated